app/models/npq_validation_data.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
-class NPQProfile < ApplicationRecord
+class NPQValidationData < ApplicationRecord
+  # TODO: Rename table
+  self.table_name = "npq_profiles"
+
+  has_one :profile, class_name: "ParticipantProfile::NPQ", foreign_key: :id
```

Comment:
We're using the primary key as a foreign key? Are the guaranteed to stay in sync for newly created records?

Reply:
Do we have any DB enforced consistency?

Reply:
Yes, they have been migrated this way and since association are defined this way it is pretty much guaranteed. Having association via id-id is quite interesting IMO as it effectively prevents reassigning the profiles (which is perfect for situations when one table is an extension of other).

However it is not really a proper RailsWay (tm) - so theoretically it is possible to reassign associated object which would in fact change the associated object id (which might be quite problematic). I'd tackle this in a separate PR.

Reply:
Will this id <-> id association lead us to not being able to delete either one?

Reply:
@tonyheadford - Etiher one of them can be deleted and we can controll the behaviour on deletion. There's really nothing special in id<->id association.

Reply:
@ethanmills There's no need for the db consistency here. Both of this record represent the same domain model (NPQ Participant) in two different bounded context (which is very common in DDD - https://martinfowler.com/bliki/BoundedContext.html) and none of them really requires the presence of the other one.